js/label/shared/Label.svelte

```diff
@@ -35,27 +42,35 @@
 			>
 				<div class="inner-wrap">
 					<meter
-						aria-labelledby="meter-text"
+						aria-labelledby={get_aria_referenceable_id(
+							`meter-text-${confidence_set.label}`
+						)}
```

Comment:
Fixed this ARIA error:
![CleanShot 2024-04-18 at 12 42 07@2x](https://github.com/gradio-app/gradio/assets/3135397/c3ec97ba-4b80-49be-8837-7fa97b63f913)